### ISSUE TITLE
Re-introduce swagger for auto-generated frontend API client code?

### ISSUE THREAD

**Issue by defaude** [qaware/wir-vs-virus#119]
Im Vorfeld von #79 haben wir uns entschieden, bis auf Weiteres swagger-codegen **nicht** zu verwenden.

Kurzer Auszug aus der vom Backend ausgegebenen swagger-Definition:

```yaml
# ...
host: 'localhost:8080'
basePath: /
# ...
paths:
  /api/admin/login:
# ...
```

Die default-Templates im Modues `typescript-angular` sind totaler Rotz. Es entstehen leider mehr oder weniger unbrauchbare client Klassen, wo die URLs eher zerschrotet sind.

Empfehlung:
* Swagger-Definition, die aus dem Backend heraus kommt, frisieren und aufhübschen (z.B. ohne host, `/api` als `basePath` usw.)
* Swagger Codegen noch einmal verproben und den generierten Code genau anschauen. Alles was nicht passt, durch angepasste Templates fixen.
* Den ganzen generierten Code nicht direkt in der Anwendung verdrahten, sondern **trotzem / genau deswegen** eine entsprechende Facade in der Anwendung einbauen.

**Comment by phxql** [comment]
Wenn wir da irgendwas im Backend anpassen sollen, sag Bescheid.

**Comment by defaude** [comment]
Jo, das Ticket hier ist erst mal als "leg auffe Halde" gedacht.
